### DISCUSSION TITLE
Is there any way to show the placeholder for lazy components again when the data is being changed

### DISCUSSION THREAD

**Discussion by robingta** [livewire/livewire#8505] [1. Help]
I have a lazy component that shows a shimmer and a collection of data but when the data is changed for if a filter action to be performed then how to show the placeholder again

**Comment by henninghorn** [comment]
I have a similar use case.

I'm rendering a list of available ressources loaded from an API.
When the component initially loads a list based on some dates I use the placeholder() method.
However, I also want to update the underlying API parameters (and make another API request) and then display the same placeholder as initially.

**Reply by robingta** [reply]
yes i mean you can not re run the placeholder think again

**Reply by Ricky-ClarkIII** [reply]
+1 on this. It would be awesome to have a way to re-trigger the placeholder when filtering tables for example.

**Reply by ghabriel25** [reply]
@Ricky-ClarkIII You could use `wire:loading` as a wrapper for your placeholder while the real content use `wire:loading.remove`

**Reply by Ricky-ClarkIII** [reply]
That works like a charm. I can't believe I didn't think of that and thank you for your suggestion!
